.github/workflows/gradle-build-pr.yml

```
@@ -57,43 +61,46 @@ jobs:
       - name: "Checkout code"
         uses: actions/checkout@v2
       - name: "Whitelist Runner IP"
-        uses: broadinstitute/datarepo-actions@0.14.0
+        uses: broadinstitute/datarepo-actions@sh-DR-1114-check-before-auth
         with:
           actions_subcommand: 'gcp_whitelist'
           role_id: ${{ secrets.ROLE_ID }}
           secret_id: ${{ secrets.SECRET_ID }}
-          k8_cluster: 'integration-master'
       - name: "Check for an availble namespace to deploy API to and set state lock"
-        uses: broadinstitute/datarepo-actions@0.14.0
+        uses: broadinstitute/datarepo-actions@sh-DR-1114-check-before-auth
         with:
           actions_subcommand: 'k8_checknamespace'
-          k8_namespaces: 'integration-1,integration-2,integration-3'
+          k8_namespaces: 'integration-1,integration-2,integration-3, integration-6'
       - name: "Build docker container via Gradle"
-        uses: broadinstitute/datarepo-actions@0.14.0
+        uses: broadinstitute/datarepo-actions@sh-DR-1114-check-before-auth
         with:
           actions_subcommand: 'gradlebuild'
       - name: "Deploy to cluster with Helm"
-        uses: broadinstitute/datarepo-actions@0.14.0
+        uses: broadinstitute/datarepo-actions@sh-DR-1114-check-before-auth
         with:
           actions_subcommand: 'helmdeploy'
           helm_secret_chart_version: '0.0.4'
           helm_datarepo_api_chart_version: '0.0.17'
```

adding chart up dates
```suggestion
          helm_datarepo_api_chart_version: '0.0.19'
```